modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -100,29 +100,16 @@ body .ddm-user-view-content {
 .lfr-ddm-form-container {
 	word-break: normal;
 
-	.btn {
-		opacity: unset;
-	}
+	.custom-control-input:disabled ~ .custom-control-label {
+		::before {
+			background-color: #0b5fff;
+			border-color: #0043c5;
```

Comment:
This is not a standard Clay color, but I think this close to `$primary-d2`, please check with Julia.